auth_backends/adfs/helsinki_library_asko.py

```diff
@@ -2,6 +2,9 @@
 
 from auth_backends.adfs.base import BaseADFS
 
+import logging
```

Comment:
This line should be close to the top of the file. Check what the `isort` command asks you to do.

Reply:
The import sorting expects imports to be in a few alphabetically sorted groups separated by blank lines: standard library imports should be first, followed by third party library imports and finally local (first party) imports.